case_studies/strong_lensing/run_sim.sh

```diff
@@ -0,0 +1,182 @@
+#!/bin/bash
+
+# Script to automate running the galsim command with the yaml configuration
+
+# Default Values
+num_files=3
+num_galaxies=3
+img_size=500
+
+
+while getopts "f:s:g:" opt; do
```

Comment:
It seem the problem was that `data/DECam_00154912_12_psfcat.psf` isn't found. Is that a small file? If so, would you add it to the repo so your script can be run by others?